cmd/gaia/app/app.go

```diff
@@ -43,17 +43,17 @@ type GaiaApp struct {
 	cdc *codec.Codec
 
 	// keys to access the substores
-	keyMain          *sdk.KVStoreKey
-	keyAccount       *sdk.KVStoreKey
-	keyStake         *sdk.KVStoreKey
+	keyMain          *sdk.IAVLStoreKey
```

Comment:
Mhmmm, why did we rename these? This assumes IAVL usage, although it probably won't change for Gaia.

Reply:
I don't want us to have this key... an IAVLStore is a KVStore... no need to create key types for specific types of KVStores.  In the future we may have other store types like HeapStores.